### DISCUSSION TITLE
Exclude already AirPlay capable devices from Addon

### DISCUSSION THREAD

**Discussion by danielschmitz87** [hassio-addons/app-airsonos#69] [Q&A] [answered]
Hi, is there a way to permanently exclude some speakers from the config?
My problem is, that I know see duplicate entries in iOS because some of my speakers have native Airplay Support.

**Comment by joohann** [comment]
I'm also looking for an answer 😊

**Comment by Ilkygul** [comment]
Any solution for this problem? I also have some old and new speakers. I want to use airplay on the old Sonos speakers but don't want duplicated airplay devises on my apple devices.

**Comment by frenck** [comment, marked as answer]
use/edit airsonos.xml that is in your config folder. Each player has an:

`<enabled>1</enabled>`

If you don't want a specific player, set it to `<enabled>0</enabled>`

**Comment by Ilkygul** [comment]
Thanks Frenck!!
